GitUI/BuildServerIntegration/BuildServerWatcher.cs

```diff
@@ -27,8 +27,7 @@ namespace GitUI.BuildServerIntegration
 {
     public sealed class BuildServerWatcher : IBuildServerWatcher, IDisposable
     {
-        private const int PollIntervalWhenSomeBuildsAreRunning = 10;
```

Comment:
Suggest keeping with a name like
CheckBuildInterval
to avoid magic numbers

Reply:
I will do.